Automattic/woocommerce-payments
# feat: add euro when euro payments enabled

Helps with https://github.com/Automattic/woocommerce-payments/issues/2170

#### Changes proposed in this Pull Request









Ensuring that when a payment method is added, if it is a EU payment method (like SEPA/Sofort/giropay), Euro should be also added in the multi-currency settings.

#### Testing instructions





- Ensure you have "Enable grouped settings" enabled
- Ensure you have "Enable UPE checkout" enabled
- Ensure you have "Enable UPE settings changes" enabled
- Ensure you have "Enable Customer multi-currency" enabled
- Ensure your store's default currency is USD
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Add one of the EU payment methods
- Save the settings
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=wcpay_multi_currency
- Euro should be added as a currency

Progress:
![Screen Shot 2021-07-15 at 5 19 04 PM](https://user-images.githubusercontent.com/273592/125865110-cc00d028-042a-40bb-aa8a-69ffb7caef3a.png)
👇
![Screen Shot 2021-07-15 at 5 19 10 PM](https://user-images.githubusercontent.com/273592/125865135-a53c4e95-25d2-4a56-b9ef-aa7d040f086c.png)
👇
![Screen Shot 2021-07-15 at 5 19 17 PM](https://user-images.githubusercontent.com/273592/125865148-504d89f1-1227-423d-a0fb-eb5707bcbf4a.png)
👇
![Screen Shot 2021-07-15 at 5 20 08 PM](https://user-images.githubusercontent.com/273592/125865156-4fb9b1a0-446e-46d4-b597-7de266928859.png)


-------------------
